🔧 Updating dependencies

Sanderand opened the pull request.

naflcosta commented on src/generator.ts:
```diff
@@ -0,0 +1,316 @@
+};
+
+const sanitizeBranch = (branch: TestBranch): TestBranch => {
+  // note: branches in duplicated conditions will be merged down one level
```
Should this commented section be left standing here?

naflcosta commented on src/globals/stats.ts:
```diff
@@ -0,0 +1,22 @@
+  skipsBecauseTested: 0,
+
+  started: Date.now(),
+};
```
Missing empty line at the end?

naflcosta commented on src/generator.ts:
```diff
@@ -0,0 +1,316 @@
+  STATS.filesRead++;
+
+  const specFilePath: string = filePath.replace('.ts', '.spec.ts');
+  const content = fs.readFileSync(filePath, 'utf-8');
```
I guess you're using multiple variables declaration convention. Otherwise please correct.

naflcosta commented on src/generator.ts:
```diff
@@ -0,0 +1,316 @@
+
+const deepTraverseConditionals = (node: ASTNode, previousIndent: string, previousConditions: Array<string>): Array<TestBranch> => {
+  const indent = previousIndent + SPACER;
+  const conditions: Array<string> = [ ...previousConditions ];
```
I guess you're using multiple variables declaration convention. Otherwise please correct.

Sanderand commented:
I've updated the dependencies and adjusted to breaking changes. Also I extracted the code out into different files without changing any of the logic.
